### PR TITLE
Separate service & worker images

### PR DESCRIPTION
**Pull request by jpopelka**
- [x] [docker hub build triggers](https://cloud.docker.com/u/usercont/repository/docker/usercont/packit-service-worker/builds)
- [x] follow up issues (#109 )
- [x] [deployment PR](https://github.com/packit-service/deployment/pull/5)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,40 +1,36 @@
+# Image for the web service (httpd), for celery worker see Dockerfile.worker
+
 FROM registry.fedoraproject.org/f29/httpd:2.4
```
Comment (TomasTomecek) [inline Dockerfile]: I hoped to have f30

File: files/install-deps-worker.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+- name: Install dependencies for packit-service worker.
+  hosts: all
+  tasks:
+  - name: Install all RPM/python packages needed to run packit-service worker.
+    dnf:
+      name:
+      - origin-clients # ? (for sandcastle ?)
+      - python3-ipdb  # for easy debugging
+      # - python3-wheel  # for bdist_wheel (where ?)
```
Comment (TomasTomecek) [inline files/install-deps-worker.yaml]: there was a check packaging target for this: feel free to scratch it

File: files/install-deps-worker.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+- name: Install dependencies for packit-service worker.
+  hosts: all
+  tasks:
+  - name: Install all RPM/python packages needed to run packit-service worker.
+    dnf:
+      name:
+      - origin-clients # ? (for sandcastle ?)
```
Comment (TomasTomecek) [inline files/install-deps-worker.yaml]: yes

File: files/install-deps-worker.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      - python3-ipdb  # for easy debugging
+      # - python3-wheel  # for bdist_wheel (where ?)
+      # - python3-kubernetes # (for sandcastle ?)
```
Comment (TomasTomecek) [inline files/install-deps-worker.yaml]: yes

File: Dockerfile (hunk 1 above)
Comment (jpopelka) [reply Dockerfile]: #110

File: Dockerfile.worker [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+RUN mkdir ${HOME} && chmod 0776 ${HOME}
+COPY files/passwd ${HOME}/passwd
+
+# Ansible doesn't like /tmp
+COPY files/install-deps-worker.yaml /src/files/
+RUN cd /src/ && \
+    ansible-playbook -vv -c local -i localhost, files/install-deps-worker.yaml \
+    && dnf clean all
+
+    # The passwd/nss_wrapper magic is needed for fedpkg
```
Comment (rpitonak) [inline Dockerfile.worker]: Just a nit, delete the indentation of comment here.

File: Makefile [hunk 6]
```diff
@@ -1,12 +1,16 @@
-TEST_TARGET := ./tests/
-PACKIT_IMAGE := docker.io/usercont/packit-service:master
+SERVICE_IMAGE := docker.io/usercont/packit-service
+WORKER_IMAGE := docker.io/usercont/packit-service-worker
```
Comment (rpitonak) [inline Makefile]: When the image described in `Dockerfile.worker.prod` is built? Is it just docker hub hook?

Comment (jpopelka) [reply Makefile]: Yes, it's just for docker hub trigger on a push to stable branch.
I haven't figured out any better way.

**Comment by rpitonak** [comment]
Feel free to merge both of them. I will leave the order in which you will do it on your decision :D

**Pull request merged**